packages/contracts-bedrock/scripts/ForgeArtifacts.sol

```diff
@@ -78,7 +78,14 @@ library ForgeArtifacts {
         string[] memory cmd = new string[](3);
         cmd[0] = Executables.bash;
         cmd[1] = "-c";
-        cmd[2] = string.concat(Executables.jq, " '.methodIdentifiers | keys' < ", _getForgeArtifactPath(_name));
+        cmd[2] = string.concat(
+            Executables.jq,
+            // We need to explicitly handle the cases where .methodIdentifiers isn't present or is
```

Comment:
Are you sure that the problem is in `Process.run`? Or in `readStringArray`? This fails if you return `[]` on empty?

Reply:
The problem is in `Process.run` and it fails if I return `[]` on empty, yep.

Reply:
I modified the function so that it clears out the dummy value.

Reply:
There is an overload of `Process.run` that takes a second arg that allows the result to be empty, see https://github.com/ethereum-optimism/optimism/blob/13661cf1e1080251beac812f2f3ffd33f188f478/packages/contracts-bedrock/scripts/libraries/Process.sol#L22